data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Sinai Syriac 2A, ff. 1-152",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "origin": {
+    "date_human-readable": "6th c. CE",
+    "date_normalized": "0500/0600",
+    "place": {
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/place/1",
+      "attested_name_NS": "ܛܘܪ ܣܝܢܝ",
+      "attested_name": "Mount Sinai",
+      "source": "Attested in the colophon on f. 152v"
+      }
+  },
+  "associated_persons":
+    [{
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/person/1",
+      "attested_name_NS": "ܝܥܩܘܒ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Ya'qub bar d-Basil",
+      "souce": "Attested in the note on f. 114v"
+    },
+    {
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/person/2",
+      "attested_name_NS": "ܒܪ ܫܒܘܢ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Bar-Shabon bar d-Basil",
+      "souce": "Attested in the note on f. 114v"
+    }],
+  "associated_places":
+  [{
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/place/2",
+      "attested_name_NS": "ܐܝܘ ܩܪܝܬܐ",
+      "attested_name": "The Village Yw",
+      "source": "Provided in connection with Jacob and Bar-Shabbon in the note on f. 114v"
+      }],
```

Comment:
Potentially add a placeholder for a IIIF content state URI for the note or name within the note. Could add this to any place we want to point to specific regions for physical evidence from an ms (e.g., exemplars of script types)

Reply:
Added a JIRA ticket for this: https://jira.library.ucla.edu/browse/SLDP-634